python/lsst/resources/s3utils.py

```diff
@@ -108,6 +111,20 @@ class _TooManyRequestsException(Exception):
 max_retry_time = 60
 
 
+def clean_test_environment(testcase: TestCase) -> None:
```

Comment:
There will need to be a docstring here that explains the purpose of this function.